src/styles/blocks/_catalog.scss

```diff
@@ -0,0 +1,40 @@
+.catalog {
+  display: grid;
+  grid-auto-flow: row;
+  justify-content: center;
+  grid-template-columns: repeat(1, $card-width);
+  gap: 48px;
+
+  &__card {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: $card-width;
+    height: $card-height;
+    border: 1px solid #ccc;
+    border-radius: 5px;
+    margin-bottom: 48px;
+  }
+
+  &__card:not(:nth-child(4n)) {
+    margin-right: 48px;
+  }
```

Comment:
You use the gap property and it sets the gaps between rows and columns. So you don't need to use margin for cards.